pyproject.toml

```diff
@@ -20,6 +20,10 @@ shylock = {extras = ["aioarangodb"], version = "^1.1.1", optional = true}
 [tool.poetry.extras]
 shylock = ["shylock"]
 
+[tool.skjold]
```

Comment:
Could you kindly move this down a bit, so that we could keep the `[tool.poetry.*]` together and not have this `[tool.skjold]` in the middle of them?